homeassistant/components/dsmr/const.py

```diff
@@ -242,41 +242,17 @@
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
     DSMRSensorEntityDescription(
-        key=obis_references.LUXEMBOURG_ELECTRICITY_USED_TARIFF_GLOBAL,
-        name="Energy Consumption (total)",
-        dsmr_versions={"5L"},
-        force_update=True,
-        device_class=SensorDeviceClass.ENERGY,
-        state_class=SensorStateClass.TOTAL_INCREASING,
-    ),
-    DSMRSensorEntityDescription(
-        key=obis_references.LUXEMBOURG_ELECTRICITY_DELIVERED_TARIFF_GLOBAL,
-        name="Energy Production (total)",
-        dsmr_versions={"5L"},
-        force_update=True,
-        device_class=SensorDeviceClass.ENERGY,
-        state_class=SensorStateClass.TOTAL_INCREASING,
-    ),
-    DSMRSensorEntityDescription(
-        key=obis_references.SWEDEN_ELECTRICITY_USED_TARIFF_GLOBAL,
+        key=obis_references.ELECTRICITY_IMPORTED_TOTAL,
         name="Energy Consumption (total)",
-        dsmr_versions={"5S"},
+        dsmr_versions={"5", "5B", "5L", "5S"},
```

Comment:
obis_reference.ELECTRICITY_IMPORTED_TOTAL was supported for:
`dsmr_versions={"2.2", "4", "5", "5B"}`
You removed support for 2.2 and 4...was that intentional?

Reply:
Yes, this was an intentional change. Those versions of the DSMR specification do not actually send out those fields. I noticed this by it not being populated on my version 4 meter.
You can verify this against https://github.com/ndokter/dsmr_parser/blob/master/dsmr_parser/telegram_specifications.py.

Reply:
I think you should mention this under breaking changes and explain that these were not actually getting any data so can be safely removed.

Reply:
Sure, updated accordingly.